.github/workflows/pr-changelog-check.yml

```diff
@@ -0,0 +1,14 @@
+name: Pull Request Changelog Check
+on: [pull_request]
+jobs:
+
+  build:
+    name: Run CI
```

Comment:
I think we should have a more descriptive name here, otherwise we'll have 2 "Run CI" workflows in the UI :)